Update 1.19.14

vectorcmdr opened the pull request:
Extra Mappings & Corvette Part Inventory / Search Hotfix
References issues #1225 #1223 #1219 #1216 #1210 

Adds mappings:
```
Cyx	Buildings
Mp7	BuildingSeeds
leA	ByteBeatJukebox
:R0	Playing
9B5	RequiresStatConversion
sus	HasScannedToReveal
t<Q	TowerPowerTimeStamps
XQ1	ClusterIndex
h8@	TimeStamps
FrH	LastWeaponRefreshTimestamp
```

Changes product category for Corvette components from ```BUILDING_PART``` to ```CUSTOMISATION_PART``` like the Starship components, restoring the ability to search via add.

Updates version strings (hopefully all?).

vectorcmdr committed:
Update 1.19.1

Extra Mappings & Corvette Part Inventory / Search Hotfix

vectorcmdr committed:
Update 1.19.1

Update zip.

vectorcmdr committed:
1.19.1

Application.class version string update.

vectorcmdr committed:
1.19.1

Application.class version string update.

vectorcmdr committed:
Update 1.19.2

Correct additional Corvette parts types.

vectorcmdr commented:
Updated additional Corvette parts to also be ```CUSTOMISATION_PART ``` type.

vectorcmdr committed:
Update 1.19.3

Update for NMS 6.03.
Corvette part stack size increased.

vectorcmdr commented:
Updated for NMS 6.03 (corvette part stack size increased).

vectorcmdr committed:
Update 1.19.4

Added second version Vesper Diffuser to Corvette parts list:

```
 ^B_STR_M_NE
 ^B_STR_M_NE1
 ^B_STR_M_NE2
 ^B_STR_M_NE3
 ^B_STR_M_NW
 ^B_STR_M_NW1
 ^B_STR_M_NW2
 ^B_STR_M_NW3
```

vectorcmdr commented:
Added the second Vesper Diffuser version per half of Issue #1241.

Parts added:
```
 ^B_STR_M_NE
 ^B_STR_M_NE1
 ^B_STR_M_NE2
 ^B_STR_M_NE3
 ^B_STR_M_NW
 ^B_STR_M_NW1
 ^B_STR_M_NW2
 ^B_STR_M_NW3
```

vectorcmdr committed:
Update 1.19.5

Added Thermal Grille Cap to Corvette parts list.

vectorcmdr commented:
Added Thermal Grille Cap inventory items ('_valid_' orientations only):
```
^B_STR_N_NE
^B_STR_N_NE1
^B_STR_N_NE2
^B_STR_N_NE3
^B_STR_N_NW
^B_STR_N_NW1
^B_STR_N_NW2
^B_STR_N_NW3
```

optikuzz commented:
Add 1.19.5 >start nmsSaveEditor 
  
Reading save file...
  Slot: Slot 2 - NORMAL - -=tion =-
  Filename: save4.hg
  WARNING: name mapping not found: kKt
  WARNING: name mapping not found: :N6
  WARNING: name mapping not found: PBQ
  WARNING: name mapping not found: 04i
  WARNING: name mapping not found: l;o
  WARNING: name mapping not found: iIi
  WARNING: name mapping not found: Eaw
  WARNING: name mapping not found: o1O
Finished.
;-(

vectorcmdr committed:
Update 1.19.6

Added Expedition 19 mappings.

vectorcmdr commented:
Added new mappings for Expedition 19 as noted in Issue #1231 
```
kKt	ForceReplaceStoryPortalSeed
:N6	SearchType
PBQ	SpecificBuildingClass
04i	UseBuildingFromRendezvousStage
l;o	NeedsDeepWaterPlanet
iIi	ExcludePlanetsWithEvents
Eaw	SystemNeedsGasGiant
o1O	NeverAllowGasGiantSystem
BFN	AltDescription
;lF	BuildableShipTechID
SuM	FossilCategory
96H	CorvettePartCategory
DTD	NeverPinnable
eSZ	FoodBonusStat
N=w	StatsType
<:g	FoodBonusStatAmount
```

vectorcmdr committed:
Update 1.19.7

Added acc mappings for experimental 09/09 in preparation for 6.05.

vectorcmdr commented:
Added acc mappings for experimental 09/09 in preparation for 6.05 per Issue #1231 :
```
>W7 CustomBindingsMac
Jxp CustomBindingsPC
MyN CustomBindingsPlaystation
jSo CustomBindingsSwitch
ilV CustomBindingsXbox
```

optikuzz commented:
After installing Update 1.19. (1 ...-... 7). I launch NMS Save Editor. The reminder pops up to upgrade to the latest version. How to disable this remark?
<img width="443" height="155" alt="20250929_193706" src="https://github.com/user-attachments/assets/93daf5d4-a2ff-4885-b17e-0484acc6438b" />

vectorcmdr commented:
You can't unless I remove it temporarily, or until the fork and main versions match. It's intentionally left.
I stated that in every post I linked it in. 

Again, stop placing issues in the pull request please.
This is bad GitHub etiquette.

vectorcmdr committed:
Update 1.19.8

Added mappings for Breach / Expedition 20.

vectorcmdr commented:
Added new mappings from libMBIN for 6.10 Breach / Expedition 20 per Issue #1265 

```
@eC	ForceBlackHolesAtPartyUAs
0Is	AltBossRewards
UMx	NeedsSpecificCreature
j=K	SystemNeedsRelicPlanet
iG6	SystemNeedsNonGasGiant
FyV	OverrideStartingUAScreenFilter
8cR	StartingUAScreenFilter
G:p	ScreenFilter
```

vectorcmdr committed:
Update 1.19.9

Add initial support for new corvette parts from base game (not expedition rewards).
Contains some ugly (but correct) placeholder images.

vectorcmdr committed:
Update 1.19.10

Added Debris Rangefinder from 6.10 (Breach) to items list.

vectorcmdr commented:
PR updated in reference to Issue #1268

Added Debris Rangefinder from 6.10 (Breach) to items list.

vectorcmdr committed:
Update 1.19.11

Added Aeron corvette parts from EXP20/6.10 (Breach) to items list.

vectorcmdr committed:
Update 1.19.12

Added rewards unlocks from EXP20/6.10 (Breach).

vectorcmdr committed:
Update 1.19.13

Fix for Aeron Trail from EXP20/6.10 (Breach)

vectorcmdr committed:
Update 1.19.14

Renamed Elixir of Blood items to add glyph number in reference to Issue https://github.com/vectorcmdr/NMSSaveEditor/issues/6

RetroGuy77 commented:
That was fast and thank you.
